tools/android_lint/bin/main.dart

```diff
@@ -0,0 +1,88 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+import 'package:process/process.dart';
+
+Future<void> main(List<String> args) async {
```

Comment:
Would it be worth adding an explanation of `args` as dartdoc to this method? Not sure what Flutter style applies to this, if anything.

Reply:
Done.